llvm_passes/core/Controller.cpp

```diff
@@ -73,8 +73,8 @@ static cl::opt<FIRegSelMethod> firegselmethod(
 static cl::opt<FIRegLoc> fireglocation(
     cl::desc("Choose fault injection register location:"), cl::init(dstreg),
     cl::values(clEnumVal(dstreg, "Inject into destination register"),
-               clEnumVal(allsrcreg,
-                         "Inject randomly into one of all source registers"),
+               clEnumVal(allreg, "Injection randomly into either destination register or one of the source registers"),
```

Comment:
Perhaps this is being pedantic, but could you please change this from 'injection' into 'inject'?